homeassistant/components/google_assistant/smart_home.py

```diff
@@ -113,6 +113,23 @@ def entity_to_device(entity: Entity):
 
 def query_device(entity: Entity) -> dict:
     """Take an entity and return a properly formatted device object."""
+    if entity.domain == climate.DOMAIN:
+        response = {
+            'thermostatMode':
+            entity.attributes.get(climate.ATTR_OPERATION_MODE),
```

Comment:
We need to add a fallback mode for `thermostatMode`. When the mode is missing, I still get the `mode not supported` error message. When hard coding it to `on`, it works as expected.

Not sure if this is the best value. Maybe someone has a better suggestions how we should deal with climate devices without operation modes?

Reply:
@sibbl fallback added. I don't think we can do better than this until #10151 is merged, standardizing the set of possible operation modes.